### PR TITLE
DPM-904: add application build configuration

### PR DESCRIPTION
**Pull request by rav-pradhan**
Additions here are to facilitate the deployment of the application to DBT Platform

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "description": "simple helpcentre cms for datahub",
   "main": "index.js",
   "scripts": {
+    "build": "echo build",
```
Comment (rav-pradhan) [inline package.json:8]: My understanding is that we need this as a workaround for the Node.js Buildpack

File: .copilot/config.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+repository: public.ecr.aws/data-hub-helpcentre
```
Comment (acodeninja) [inline .copilot/config.yml:1]:
```suggestion
repository: dshc
```

You'll want to update this before merging.

File: .tool-versions [hunk 3]
```diff
@@ -0,0 +1 @@
+python 3.12.6
```
Comment (acodeninja) [inline .tool-versions:1]: Probably don't need to add this.

File: .copilot/phases/pre_build.sh [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+    sed -i "s|url = git@github.com:uktrade/\(.*\).git|url = $git_clone_base_url/\1.git|g" ./.gitmodules
+
+    git submodule update --init --remote --recursive
+
+else
+  echo ".gitmodules file does not exist. No URLs to update."
+fi
+
+# Add commands below to run as part of the pre_build phase
```
Comment (acodeninja) [inline .copilot/phases/pre_build.sh:23]:
```suggestion
# Add commands below to run as part of the pre_build phase

python manage.py compilescss
python manage.py collectstatic --noinput
```

You'll want to add these commands in here to build the assets.

**Review by acodeninja — COMMENTED** [review]
Few updates before merge.

**Pull request merged**